langchain/memory/chat_message_histories/file.py

```diff
@@ -28,6 +28,13 @@ def __init__(self, file_path: str):
         if not self.file_path.exists():
             self.file_path.touch()
             self.file_path.write_text(json.dumps([]))
+        else:
+            try:
+                # Try to load the content and parse it with json
+                json.loads(self.file_path.read_text())
+            except json.JSONDecodeError:
+                # If it fails, initialize the file with an empty list
+                self.file_path.write_text(json.dumps([]))
```

Comment:
so this is overwriting the original file? would it be better to create a new file?